src/tactic/interval_cases.lean

```diff
@@ -188,9 +188,16 @@ an `Ico` interval corresponding to a lower and an upper bound.
 
 Here `hl` should be an expression of the form `a ≤ n`, for some explicit `a`, and
 `hu` should be of the form `n < b`, for some explicit `b`.
+
+`n` if provided is the name of the hypothesis added. If an automatically generated name is wanted,
+pass in `none` as `n`.
```

Comment:
By default `interval_cases_using` automatically generates a name for the new hypothesis. The name can be specified via the optional argument `n`.